pkgs/development/python-modules/zigpy-cc/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, buildPythonPackage, fetchPypi
+{ python, stdenv, buildPythonPackage, fetchPypi
```

Comment:
What's the purpose of this change?

Reply:
I was trying to add testing, but Pypi tarballs don't have the tests, and I just forgot to remove that afterwards, it's fixed now.

Reply:
I was really expecting tests to run, since this has checkInputs set. Can you move to `fetchFromGitHub` instead? We need either working tests (preferred) or `pythonImportTests`.